testing/upload_validation.ts

```diff
@@ -832,6 +832,7 @@ const genericObjectSchema: z.ZodTypeAny = z.lazy(() =>
     attribution: attributionSchema,
     properties: z.record(objectPropertyUnionSchema),
     includeUnknownProperties: z.boolean().optional(),
+    __packId: z.number().optional(),
```

Comment:
Is there a way to validate that this is NOT provided? I think that should work--we don't want anybody ever actually specifying this in a pack definition, it will only ever be injected by us after they upload and we validate the schema.

Reply:
Ok. We may want to throw on the `coda` side if we detect this just to be safe--we'll be overwriting it, so maybe we don't have to throw, but might be best to warn the maker and just ensure it's not being abused.

TS won't allow somebody to provide this field in non-legacy functions, but if a maker isn't using TS or abusing TS they can still upload whatever they want, and indeed we got a 500 error in the diff checker that seems to have resulted from somebody bypassing TS and us not accounting for that.

Reply:
Awesome, I'll add that to https://github.com/coda/coda/pull/73515